docs/unix/quickref.rst

```diff
@@ -0,0 +1,93 @@
+.. _unix_quickref:
+
+Quick reference for the UNIX and Windows ports
+==============================================
+
+Command line options
+--------------------
+
+Usage::
+
+    micropython [ -h ] [ -i ] [ -O<level> ] [ -v ] [ -X <option> ] [ -c <command> | -m <module> | <script> ] [ <args> ]
+
+
+Invocation options:
+
+.. option:: -c <command>
+
+   Runs the code in ``<command>``. The code can be one or more Python statements.
+
+.. option:: -m <module>
+
+   Runs the module ``<module>``. The module must be in ``sys.path``.
+
+.. option:: <script>
+
+   Runs the file ``<script>``. The file can be a plain text ``.py`` file or
+   a compiled ``.mpy`` file.
+
+If none of the 3 options above are given, then MicroPython is run in an
+interactive REPL mode.
+
+
+.. option:: <args>
+
+    Any additional arguments after the command, module or script will be passed
+    to `sys.argv`. This is currently only implemented for the module option.
```

Comment:
`sys.argv` will work also for scripts passed without `-m`, eg `micropython foo.py arg1 arg2`

Reply:
Fixed.